terminus-ui/expansion-panel/src/expansion-panel.component.ts

```diff
@@ -297,10 +303,8 @@ export class TsExpansionPanelComponent extends CdkAccordionItem implements After
     // istanbul ignore else
     if (this.lazyContent) {
       // Render the content as soon as the panel becomes open.
-      this.opened.pipe(
-        // tslint:disable: no-non-null-assertion
-        startWith<void>(null!),
-        // tslint:enable: no-non-null-assertion
+      scheduled([[of(null)], this.opened], asap).pipe(
+        concatAll(),
```

Comment:
oh I like this... is this new in rxjs update?

Reply:
Yep, there's actually two ways to solve this. Basically they deprecated the version of `startWith` that has only a single type. So we can use `startWith<null, void>` (which I did somewhere). But the other refactor was using `scheduled` which I had never used so I thought I'd try it. Both seem pretty logical to me

Reply:
Example of the other version:

<img width="432" alt="Screen Shot 2019-06-21 at 9 29 30 AM" src="https://user-images.githubusercontent.com/270193/59925908-1d5fa480-9407-11e9-81ea-4fb581c6c3de.png">

Reply:
New definition:

```typescript
export function startWith<T, D>(v1: D): OperatorFunction<T, T | D>;
```